segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/DurableLog.java

```diff
@@ -329,9 +329,9 @@ public boolean isOffline() {
         log.debug("{}: Queuing MetadataCheckpointOperation.", this.traceObjectId);
         MetadataCheckpointOperation op = new MetadataCheckpointOperation();
         return this.operationProcessor
-                .process(op, OperationPriority.Normal)
+                .process(op, OperationPriority.SystemCritical)
                 .thenApply(v -> {
-                    log.info("{}: MetadataCheckpointOperation durably stored.", this.traceObjectId);
+                    log.info("{}: MetadataCheckpointOperation({}} stored.", this.traceObjectId, op.getSequenceNumber());
```

Comment:
Something seems wrong with the log parameter: instead of `({}}` you may want to use `({})`

Reply:
fixed.